lib/beaker/command.rb

```diff
@@ -72,9 +72,13 @@ def initialize command, args = [], options = {}
     def cmd_line host, cmd = @command, env = @environment, pc = @prepend_cmds
       env_string = host.environment_string( env )
       prepend_commands = host.prepend_commands( cmd, pc, :cmd_exe => @cmdexe )
+      if host[:platform] =~ /cisco_nexus/ && host[:user] != 'root'
```

Comment:
Doesn't this go in the opposite direction of #1051 / #1062 by putting platform-specific logic back into this module?